src/engine/command.cpp

```diff
@@ -1028,8 +1028,32 @@ static inline void skipcomments(const char *&p)
     for(;;)
     {
         p += strspn(p, " \t\r");
-        if(p[0]!='/' || p[1]!='/') break;
-        p += strcspn(p, "\n\0");
+        if(p[0] == '/' && p[1] == '/')
+        {
+            p += strcspn(p, "\n\0");
+        }
+        else if(p[0] == '/' && p[1] == '*')
+        {
+            p += 2;
+            for(;;)
+            {
+                p += strcspn(p, "*\0");
```

Comment:
I'm confused. How can a null byte (which I think `\0` represents) be useful here? The second parameter is already a null-terminated string. The array equivalent of that string should be `['*', 0, 0]` as far as I can see... Does this have any effect at all? If yes, I think you'll have to document it in a comment.

Reply:
Haha, I didn't think about that. Good catch. The old code was doing it like this too.